test/verify/check-pages

```diff
@@ -633,6 +633,16 @@ OnCalendar=daily
         b.key_press("stuff/")
         b.wait_in_text("#file-autocomplete-widget li:nth-of-type(5) button", "other")
 
+        # test pre-selected autocomplete widget
+        b.wait_val("div#demo-file-ac-preselected input", "/home/admin/.bashrc")
+        b.click("div#demo-file-ac-preselected input")
+        b.wait_in_text("#file-autocomplete-widget-preselected li:nth-of-type(1) button", "/home/admin/")
```

Comment:
For loop over the files array?

Reply:
@KKoukiou While running the VM again to write the for loop, there was no `.bash_history` file (even `ssh`ed into the VM to check), so I removed it from the test from now.

I'm not sure if the rest of the files will always be present, otherwise it might be better to check a different folder

Reply:
I think the best is to use a folder that your test creates, which has files that you test also creates. To be completely distro indepedent. Cockpit tests run on multiple distros, and for example on debian bash is not the default shell.